Configure

```diff
@@ -1110,8 +1110,9 @@ $target{exe_extension}=".exe" if ($config{target} eq "DJGPP"
 $target{exe_extension}=".pm"  if ($config{target} =~ /vos/);
 
 ($target{shared_extension_simple}=$target{shared_extension})
-    =~ s|\.\$\(SHLIB_VERSION_NUMBER\)||;
-$target{dso_extension}=$target{shared_extension_simple};
+    =~ s|\.\$\(SHLIB_VERSION_NUMBER\)||
+    unless defined($target{shared_extension_simple});
```

Comment:
Awright, I hear you, but I want it duly noted that if someone writes a config where `shared_extension_simple` contains a shlib version number, that defeats the spirit if that variable (i.e. "simple" is out the window)

Reply:
We have different ideas in this case, but it's not important enough to discuss further, at least for the moment.  You have my approval, go merge ;-)

Reply:
> go merge ;-)

I won't. It's WIP, I'm waiting for feedback from user, and there will be more modifications, to install procedure.

Reply:
Ah right, I forgot for a moment.  So I'll wait